assets/app/scripts/controllers/newfromtemplate.js

```diff
@@ -1,4 +1,5 @@
 'use strict';
+/* jshint unused: false */
```

Comment:
I think that function got refactored to `Navigate.toErrorPage` and never removed